src/main/java/com/mjcode/model/Game.java

```diff
@@ -0,0 +1,268 @@
+package com.mjcode.model;
+
+import java.awt.*;
+import java.util.ArrayList;
+
+public class Game {
+
+    // Declare fields
+    private Player player1;
+    private Player player2;
+    private Player currentPlayer;
+    private ArrayList<Move> moves;
+    private boolean gameStart;
+    private Tile[][] board;
+
+
+    /**
+     * No-args constructor initializes the fields
+     */
+    public Game() {
+        player1 = null;
+        player2 = null;
+        currentPlayer = null;
+        gameStart = false;
+        moves = new ArrayList<>();
+        board = new Tile[8][8];
+    }
+
+    /**
+     *  If the game has no players, sets the supplied Player to be Player One.
+     *  If the game currently has one player, sets the supplied Player to be Player Two,
+     *  assigns colors and directions to each Player as appropriate, and starts the game.
+     *  If the game currently has two non-null players, does nothing.
+     * @param newPlayer To add in the game
+     */
+    public void addPlayer(Player newPlayer) {
+        // If game has no player
+        if (player1 == null && player2 == null) {
+            player1 = newPlayer;
+            player1.setColor(Color.WHITE);
+            player1.setDirection(1);
+        }
+
+        // If game currently has one player
+        else if (player1 != null && player2 == null) {
+            player2 = newPlayer;
+            player2.setColor(Color.BLACK);
+            player2.setDirection(-1);
+            gameStart = true;
+        }
+    }
+
+    /**
+     * The endGame method returns true if either Player has no more Pieces
+     * @return True if player has no more pieces
+     */
+    protected boolean endGame() {
+        return player1.getPieces().isEmpty() || player2.getPieces().isEmpty();
+    }
+
+    /**
+     * The getTile method returns the Tile on the board at the specified row and column.
+     * @param row On the board
+     * @param column on the board
+     * @return The tile
+     */
+    public Tile getTile(int row, int column) {
+        return board[row][column];
+    }
+
+    /**
+     * The executeMove method updates the Pieces on the Board to reflect the Move specified, and informs
+     * both Players that the Move has been executed via their respective executeMove methods. If the Move
+     * results in a capture, removes the captured Piece from the board and from the appropriate
+     * Player's list of Pieces. If the Move results in a Piece occupying its opponent Player's
+     * first row*, the Piece is kinged. Finally, adds the specified Move to the history of Moves made thus far.
+     * @param move The move
+     */
+    protected void executeMove(Move move) {
+        // Declare and initialize the objects
+        Tile fromTile = move.getFromTile();
+        Tile toTile = move.getToTile();
+        Piece movedPiece = move.getMovedPiece();
+        Piece capturedPiece = move.getCapturedPiece();
```

Comment:
Create a new method in Game class that returns the tile being jumped by the move.

Then in executeMove, get the captured piece from the jumped tile's occupant.